### DISCUSSION TITLE
Can I use DragSelect with Angular 17?

### DISCUSSION THREAD

**Discussion by Biowavix** [ThibaultJanBeyer/DragSelect#216] [Q&A]
I am looking for a drag-and-select library to use in an Angular 17 application, but to my surprise, I have not found any library that is kept up to date.

After a deeper search, I found DragSelect, which seems very well made. However, I'm not exactly sure how to adapt this library to use with Angular. Before I start researching how to do it and potentially do it wrong, has anyone done this before?

Thank you in advance!

**Comment by ThibaultJanBeyer** [comment]
Thanks @Biowavix it should work with Angular! I am wondering, did you give it a try and was able to set it up? Thank you, kind regards, Thibault
